static/docs/get-started/index.md

```diff
@@ -24,3 +24,6 @@ motivation and what's happening internally.
 You'll need [Git](https://git-scm.com) to run the commands in this tutorial.
 Also, if DVC is not installed, please follow these [instructions](/doc/install)
 to do so.
+
+▶ For a quick start you can also
```

Comment:
that's sad, I though they would be rendered as here https://unicode.org/emoji/charts/full-emoji-list.html

not sure why are they not

Reply:
let's make it a small black triangle or figure out how to render it properly

Reply:
Let's not try to make it perfect since it is only temporary and it will be replaced by the button (#670).